src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="img__logo">
+        <img
+          src="./images/logo.png"
+          alt="Moto logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a href="#" class="is-active link__item">Apple</a>
+          </li>
+          <li class="list__item">
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="is-active nav__item">Apple</a>
          </li>
          <li class="nav__item">
```

it is not critical now(u didn't study BEM yet) but it is more meanfull names, so fix it every where
also add meanfull hrefs